src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,7 @@
+.calendar {
+  @each $name, $number in $week-day {
```

Comment:
Ensure that the variable `$week-day` is defined and accessible in this context. It should be a map or list that provides the necessary data for the `@each` loop to function correctly.